integration_test/integration_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Fail on any error.
+set -eo pipefail
+
+# Display commands being run.
+set -x
+
+SERVICE_KEY="${KOKORO_KEYSTORE_DIR}/72935_cloud-profiler-e2e-service-account-key"
+COMMIT=$(git rev-parse HEAD)
+
+export GCLOUD_TESTS_NODEJS_PROJECT_ID="cloud-profiler-e2e"
+export GCLOUD_TESTS_NODEJS_ZONE="us-west1-a"
+export GOOGLE_APPLICATION_CREDENTIALS="${SERVICE_KEY}"
+
+# Move test to go path.
+export GOPATH="$HOME/go"
+mkdir -p "$GOPATH/src"
+cp -R "git/cloud-profiler-nodejs/integration_test" "$GOPATH/src"
+
+# Run test.
+cd "$GOPATH/src/integration_test"
+go get -d -t ./
+go test -timeout=30m -parallel=5 -tags=integration -run TestAgentIntegration -commit="$COMMIT"
```

Comment:
You don't need the '-tags=integration' as integration_test.go doesn't use build tags. Also change to '-parallel=2' as there is only 2 parallel tests?

Reply:
Thanks! -- Done.